src/main/src/main/java/org/geoserver/catalog/impl/CatalogImpl.java

```diff
@@ -2277,4 +2246,12 @@ public <T extends CatalogInfo> T get(Class<T> type, Filter filter)
     public CatalogCapabilities getCatalogCapabilities() {
         return facade.getCatalogCapabilities();
     }
+
+    public CatalogImpl getRawCatalog() {
```

Comment:
Bit of Javadoc with an explanation of why the original CatalogImpl is not "raw"?